### PR TITLE
Add `:remove-multiple-non-indenting-spaces?` usage to README

### PR DESCRIPTION
**Pull request by mpenet**
follow up to https://github.com/weavejester/cljfmt/pull/203 

This adds usage to the readme, trying to pave the way for a potential release.

### CODE REVIEW

**Comment by weavejester** [comment]
Thanks for the commit. Can you remove the additional edits to ending whitespace from the patch, and change the commit message to:

```
Add 'remove non-indenting spaces' option to README
```

**Comment by mpenet** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -136,6 +136,10 @@ selectively enabled or disabled:
   true if cljfmt should break hashmaps onto multiple lines. This will
   convert `{:a 1 :b 2}` to `{:a 1\n:b 2}`. Defaults to false.
 
+*`:remove-multiple-non-indenting-spaces?` - true if cljfmt should
```
Comment (weavejester) [inline README.md:139]: You're missing a space between the `*` and the \`. I'm not sure if the markdown parser will like that. Also can you use the same formatting as the other bullet points? So a newline after the `-`.

Comment (mpenet) [reply README.md]: should be good now

**Comment by weavejester** [comment]
The word wrapping is off, but I'll add the docs in myself.

**Pull request closed without merging**